src/builtins.rs

```diff
@@ -167,12 +179,89 @@ lazy_static! {
                     }
                 }
             }
+        ),
+        (
+            "EXPT",
+            BuiltIn {
+                decl : "FUNCTION EXPT<U : ANY_NUM, V: ANY_NUM> : U
+                VAR_INPUT
+                    ELEMENT: U;
+                    EXPONENT: V;
+                END_VAR
+                END_FUNCTION
+                ",
+                annotation: Some(|annotator, operator , parameters, ctx| {
+                    let params = parameters.ok_or_else(|| Diagnostic::codegen_error("EXPT requires parameters", operator.get_location()))?;
+                    if let [element, exponant] = flatten_expression_list(params)[..] {
+                        //Resolve the parameter types
+                        let element_type = annotator.annotation_map.get(element).and_then(|it| get_type_for_annotation(annotator.index, it));
```

Comment:
is annotationmap.get_type already offering the same?

Reply:
Looks like it is, changed now